fatal error: 'source_location' file not found with clang on AIX

Despite running tests builds (which passed) after merging https://github.com/nodejs/build/pull/4286 we have begun to see the following error when building the `main` branch with clang:
```
18:04:57 In file included from ../deps/v8/src/libplatform/default-foreground-task-runner.cc:5:
18:04:57 In file included from ../deps/v8/src/libplatform/default-foreground-task-runner.h:11:
18:04:57 In file included from ../deps/v8/include/libplatform/libplatform.h:11:
18:04:57 In file included from ../deps/v8/include/libplatform/v8-tracing.h:14:
18:04:57 In file included from ../deps/v8/include/v8-platform.h:17:
18:04:57 ../deps/v8/include/v8-source-location.h:9:10: fatal error: 'source_location' file not found
18:04:57     9 | #include <source_location>
18:04:57       |          ^~~~~~~~~~~~~~~~~
```

Further tests have shown this happens on
- [test-ibm-aix72-ppc64_be-1](https://ci.nodejs.org/computer/test%2Dibm%2Daix72%2Dppc64%5Fbe%2D1/) 
- [test-ibm-aix72-ppc64_be-3](https://ci.nodejs.org/computer/test%2Dibm%2Daix72%2Dppc64%5Fbe%2D3/)

but not on [test-ibm-aix72-ppc64_be-2](https://ci.nodejs.org/computer/test%2Dibm%2Daix72%2Dppc64%5Fbe%2D2/) (where the testing was done)

--- a/ansible/MANUAL_STEPS.md
+++ b/ansible/MANUAL_STEPS.md
@@ -523,13 +523,13 @@ runtime:
 
 utilities:
 
-1. Download the current *.tar.Z from https://www.ibm.com/support/pages/ibm-open-xl-cc-utilities-aix-1712#DNLD
+1. Download the current *.tar.Z from https://www.ibm.com/support/pages/ibm-open-xl-cc-utilities-aix-1713
 2. scp tar onto the target
 3. On the target
 
   ```sh
-  uncompress IBM_OPEN_XL_CPP_UTILITIES_17.1.2.8_AIX.tar.Z
-  tar -xf IBM_OPEN_XL_CPP_UTILITIES_17.1.2.8_AIX.tar
+  uncompress IBM_OPEN_XL_CPP_UTILITIES_17.1.3.0_AIX.tar.Z
+  tar -xf IBM_OPEN_XL_CPP_UTILITIES_17.1.3.0_AIX.tar
   inutoc .
   installp -aFXYd . ALL
   ```

